changelog-generator/content/changelogs/20231027_data_service-1-3-0.md

```diff
@@ -0,0 +1,14 @@
+---
+service: data-service
+version: 1.3.0
+date: 2023-10-27
+---
+
+**🎁 Features**
+* Disable the total count in the listing query
+    * Added a skipCount() RQL function that instructs the listing function not to execute and return the total count.
+
+
+**🐞 Bugs Fixed**
+* Known `SERVICE_EXCEPTION`s thrown on RQL errors are now resolved to `INVALID_RQL_EXCEPTION`s
+* Double encoding the `<` and `>` characters when searching for them now works.
```

Comment:
The `<` and `>` characters may now be used in RQL when all values double encoded.

Not sure about this one? 🤔